src/main/java/cpsc/dlsproject/ast/Expressions/BinaryOperation.java

```diff
@@ -0,0 +1,76 @@
+package cpsc.dlsproject.ast.Expressions;
+
+public class BinaryOperation extends Expression{
+    /**
+     * The operator of the binary operation.
+     */
+    public BinaryOperator operator;
+
+    /**
+     * The left hand side operand.
+     */
+    public Expression lhs;
+
+    /**
+     * The right hand side of the operand.
+     */
+    public Expression rhs;
+
+    public BinaryOperation(String oper) {
+        switch (oper) {
+            case "PLUS" :
```

Comment:
Make sure to be consistent here between using named symbols (e.g. `PLUS`, `MULTI`) vs. mathematical symbols (e.g. `-`, `>`).